flink-kubernetes/src/main/java/org/apache/flink/kubernetes/configuration/KubernetesConfigOptions.java

```diff
@@ -440,6 +441,28 @@ public class KubernetesConfigOptions {
                                             code("FlinkKubeClient#checkAndUpdateConfigMap"))
                                     .build());
 
+    public static final ConfigOption<Duration> KUBERNETES_TRANSACTIONAL_OPERATION_INITIAL_RETRY_DEALY =
+            key("kubernetes.transactional-operation.initial-retry-delay")
+                    .durationType()
+                    .defaultValue(Duration.ofMillis(5000))
```

Comment:
I think the 5s default initial delay might be too large. I'd suggest maybe 50ms.

Reply:
That also means it would take 12 retries to reach the default 1m max delay. Maybe we can also increase the default value for max-retires to maybe 15 or 20 times?